<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prefs_Display</class>
 <widget class="QWidget" name="Prefs_Display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Display</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Pages</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="showRulersCheckBox">
         <property name="text">
          <string>Show Rulers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showRulersRelativeToPageCheckBox">
         <property name="text">
          <string>Show Rulers Relative to Page</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showFramesCheckBox">
         <property name="text">
          <string>Show Frames</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showImagesCheckBox">
         <property name="text">
          <string>Show Images</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showTextChainsCheckBox">
         <property name="text">
          <string>Show Text Chains</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showControlCharsCheckBox">
         <property name="text">
          <string>Show Text Control Characters</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showLayerIndicatorsCheckBox">
         <property name="text">
          <string>Show Layer Indicators</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showUnprintableAreaInMarginColorCheckBox">
         <property name="text">
          <string>Show Unprintable Area in Margin Color</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showBleedAreaCheckBox">
         <property name="text">
          <string>Show Bleed Area</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showPageShadowCheckBox">
         <property name="text">
          <string>Show Page Shadow</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showVerifierWarningsOnCanvasCheckBox">
         <property name="text">
          <string>Show Preflight Verifier Warnings on the Page</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>4</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Scratch Space</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Left:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ScrSpinBox" name="scratchSpaceLeftSpinBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Right:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ScrSpinBox" name="scratchSpaceRightSpinBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Top:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="ScrSpinBox" name="scratchSpaceTopSpinBox"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Bottom:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="ScrSpinBox" name="scratchSpaceBottomSpinBox"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Gaps Between Pages</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Horizontal:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ScrSpinBox" name="pageGapHorizontalSpinBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Vertical:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="ScrSpinBox" name="pageGapVerticalSpinBox"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>195</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Display Scaling</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSlider" name="adjustDisplaySlider">
           <property name="minimum">
            <number>-150</number>
           </property>
           <property name="maximum">
            <number>500</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickInterval">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="displayDPI">
           <property name="text">
            <string>Scale%</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="buttonRestoreDPI">
           <property name="toolTip">
            <string>Resets the scale to the default dpi of your display</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="displaySizeRuler">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Sunken</enum>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="rulerUnitComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>263</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Colors</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>212</width>
            <height>292</height>
           </rect>
          </property>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Page Fill:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pageFillColorButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Scratch Space:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="scratchSpaceColorButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Selected Page Border:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="selectedPageBorderButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Frames:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QPushButton" name="frameColorButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Locked Frames:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="frameLockedColorButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Selected Frames:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QPushButton" name="frameSelectedColorButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_31">
             <property name="text">
              <string>Linked Frames:</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="frameLinkedColorButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>Grouped Frames:</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QPushButton" name="frameGroupedColorButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>Annotation Frames:</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QPushButton" name="frameAnnotationColorButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_32">
             <property name="text">
              <string>Text Control Characters:</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QPushButton" name="textControlCharsButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">ui/scrspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
